### PR TITLE
Freeze status 

### PR DESCRIPTION
**Pull request by henryyang42**
Change status implementation
- Exclude all invisible submissions then do paging
- Fix owner coowner view code bug

### CODE REVIEW

**Comment by nickelcheng** [comment]
This seems to be good for me.
@bruce3557 please check

File: status/templatetags/status_filters.py [hunk 1]
```diff
@@ -143,7 +91,8 @@ def show_detail(submission, user):
         return True
     # a user can view his team member's detail
     if submission.team:
-        team_member = TeamMember.objects.filter(team=submission.team, member=user)
+        team_member = TeamMember.objects.filter(
+            team=submission.team, member=user)
```
Comment (bruce3557) [inline status/templatetags/status_filters.py]: not sure why we need break lines here?

Comment (henryyang42) [inline status/templatetags/status_filters.py]: It will exceed 80 char in an line.

**Comment by CHIAHSINHOU** [comment]
This PR is ok for me.

**Comment by bruce3557** [comment]
+1

**Pull request merged**